package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "radiance-ui",
-  "version": "12.4.0",
+  "version": "13.0.0-alpha.3",
```

Comment:
This will get changed to 13 once we release in `master` anyway. Don't feel like another commit build for it.